src/db.c

```diff
@@ -308,6 +308,57 @@ static const char *sort_clause[] =
     "ORDER BY shuffle_pos ASC",
   };
 
+
+/* Keep in sync with enum media_kind */
+static char *media_kind_label[] = { "music", "movie", "podcast", "audiobook", "musicvideo", "tvshow" };
```

Comment:
There is too high risk that someone (maybe myself...) will change enum media_kind and not be aware of this. So the "keep in sync" comment needs to be both places, but actually I would suggest moving the label array to the header, right next to the enum. Makes it easier to maintain.